packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.tsx

```diff
@@ -21,9 +21,9 @@ export class BaseAutoFill extends BaseComponent<IBaseAutoFillProps, IBaseAutoFil
     enableAutoFillOnKeyPress: [KeyCodes.down, KeyCodes.up]
   };
 
-  private _inputElement: HTMLInputElement;
-  private _autoFillEnabled: boolean = true;
-  private _value: string;
+  protected _inputElement: HTMLInputElement;
```

Comment:
The correct way to do this in React is through composition rather than exposing internals in a base class.